frontends/web/src/routes/new-settings/components/appearance/activeCurrenciesDropdownSetting.tsx

```diff
@@ -11,13 +12,15 @@ type SelectOption = {
 
 type TSelectProps = {
   options: SelectOption[];
-  selectedCurrencies: SelectOption[];
-}
+} & Omit<SharedProps, 'btcUnit'>
```

Comment:
I was quickly confused by this, alternative maybe make `btcUnit` optional in SharedProps?

but I am not sure, but TypeScript seems to be happy. Anyway this way seems to be good too.

Reply:
Just didn't want to mess with "old" code and making btcUnitan optional - worried of breaking something. But seems like it's fine. I agree, making btcUnit optional will most likely be clearer at first glance. Will do this :)  Thanks!